core/src/test/java/org/elasticsearch/indices/stats/IndexStatsIT.java

```diff
@@ -692,7 +703,7 @@ public void testFlagOrdinalOrder() {
     }
 
     public void testMultiIndex() throws Exception {
-        assertAcked(prepareCreate("test1").setSettings("index.mapping.single_type", false));
+        assertAcked(prepareCreate("test1").setSettings("index.version.created", Version.V_5_6_0.id));
```

Comment:
Can you add a comment about why it is important that we have indices with multiple types?